apps/users/forms.py

```diff
@@ -20,8 +20,8 @@
 logger = logging.getLogger(__name__)
 
 CAPTCHA_HELP = _(
-    "Solve the math problem and click on the correct result.<br>"
-    "If you are having difficulty please contact us  by {}email{}."
+    "Solve the math problem and click on the correct result.<strong>"
+    "If you are having difficulty please contact us  by {}email{}.</strong>"
```

Comment:
hmm, as this is all a string, including the tags, does it need translation too? @goapunk

Reply:
yes, once merged I will update translations and push them to transifex